charts/tekton-apps/templates/general/argocd_applications.yaml

```diff
@@ -20,6 +20,9 @@ metadata:
   namespace: argo-cd
   annotations:
     argocd.argoproj.io/sync-wave: {{ $argocd.syncWave | default "210" | quote }}
+    {{- if $project.argocd.notifications }}
+    jiraURL: {{ $project.jiraURL }}
```

Comment:
The same situation with `jiraURL` as above. Also why here is no output for `$project.argocd.notifications.annotations` itself?

Reply:
Because adding the subscription in the AppProject pass them to all its child applications


See: https://argo-cd.readthedocs.io/en/latest/operator-manual/notifications/subscriptions/

Reply:
@luciano-buono so why do we need this `{{- if $project.argocd.notifications }}` then?